### ISSUE TITLE
Question about collaboration invitation

### ISSUE THREAD

**Issue by qwerty541** [R5-3600/R5-3600#1]
Hey, @R5-3600! Recently I received from you an invitation to collaborate on https://github.com/R5-3600/github-readme-stats. I want to know did you sent it by mistake or this action was intentional?

I'm writing it there since issues in https://github.com/R5-3600/github-readme-stats was disabled and I didn't found an email in your profile.

**Comment by R5-3600** [comment]
Hi sorry, just saw this now. I was trying to get the automatic pull bot to pull changes from the original repo to work but it was complaining about how your original repo changes weren't authorized. I was reading about it, it mentioned to add you as a collaborator. Unless do you know anyway to get the pull bot to work correctly? At first, it worked fine and all changes from the original repo were pulled right away. Now it doesn't, it gets backed up by weeks at a time because I don't check it often and it complains about authorization. Thanks.

**Comment by qwerty541** [comment]
I got a discussion about this problem with another user a few months ago. You can check it there https://github.com/anuraghazra/github-readme-stats/discussions/3468.

TL;DR You have to disable git fork protection and pull requests deployment in Vercel settings.

Feel free to contact me if you have any more questions.

**Comment by R5-3600** [comment]
> I got a discussion about this problem with another user a few months ago. You can check it there [anuraghazra/github-readme-stats#3468](https://github.com/anuraghazra/github-readme-stats/discussions/3468).
> 
> TL;DR You have to disable git fork protection and pull requests deployment in Vercel settings.
> 
> Feel free to contact me if you have any more questions.

Thank you. I was able to disable the git fork protection but I couldn't find anything about pull requests deployment inside of the Vercel settings.

**Comment by qwerty541** [comment]
> > I got a discussion about this problem with another user a few months ago. You can check it there [anuraghazra/github-readme-stats#3468](https://github.com/anuraghazra/github-readme-stats/discussions/3468).
> > TL;DR You have to disable git fork protection and pull requests deployment in Vercel settings.
> > Feel free to contact me if you have any more questions.
> 
> Thank you. I was able to disable the git fork protection but I couldn't find anything about pull requests deployment inside of the Vercel settings.

Dashboard => Project => Settings => Git => Ignored Build Step => Only build production

**Comment by R5-3600** [comment]
> > > I got a discussion about this problem with another user a few months ago. You can check it there [anuraghazra/github-readme-stats#3468](https://github.com/anuraghazra/github-readme-stats/discussions/3468).
> > > TL;DR You have to disable git fork protection and pull requests deployment in Vercel settings.
> > > Feel free to contact me if you have any more questions.
> > 
> > 
> > Thank you. I was able to disable the git fork protection but I couldn't find anything about pull requests deployment inside of the Vercel settings.
> 
> Dashboard => Project => Settings => Git => Ignored Build Step => Only build production

Thank you
